package.json

```diff
@@ -9,19 +9,30 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "^2.0.1",
+    "@supabase/auth-helpers-nextjs": "^0.8.7",
+    "@supabase/supabase-js": "^2.39.0",
+    "next": "14.0.4",
+    "next-redux-wrapper": "^8.1.0",
     "react": "^18",
     "react-dom": "^18",
-    "next": "14.0.4"
+    "react-redux": "^9.0.4",
+    "redux": "^5.0.0",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^3.1.0",
```

Comment:
Possible to remove these dependencies since we're not going to use redux?

Reply:
✅